talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>The way I am doing this code review is the following: I use the ReviewNB interface where I can always only comment on one cell as a whole (unless I am overseeing a functionality - that is not unlikely). In that case, it would be very handy if text and code would be split into smaller units per cell in the future.</p><p> </p><p>I am open to suggestions for more efficient code review strategies for Jupyter Notebooks, if you have some for me.</p><p> </p><p>Edit to the text:</p><blockquote>sidechains &gt; side chains</blockquote><blockquote>proteins.plus or Proteins.plus (decide on one spelling)</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>